java-vmwareengine/README.md

```diff
@@ -0,0 +1,201 @@
+# Google Google Cloud VMware Engine Client for Java
```

Comment:
Typo

Reply:
I coped the `name_pretty` value in DRIFT: 
![image](https://user-images.githubusercontent.com/6621793/204618198-65fa9e8b-b893-4457-ae11-7b139d214aa6.png)

Reply:
I copied the `display name` value on drift:
![image](https://user-images.githubusercontent.com/6621793/204618639-3daee6a7-8b12-4451-b8de-e81d94c16583.png)

Should I change it to `Cloud VMware Engine`?

Reply:
Can you change this line "Google Cloud VMware Engine Client for Java"? No need to re-run the script.

Reply:
Wouldn't it get overwritten the next time owlbot post processor runs? I thought the README template is from: https://github.com/googleapis/synthtool/blob/934c51d74f1ec59a20892525248de1bed8fec638/synthtool/gcp/templates/java_library/README.md?plain=1#L5

Reply:
Ok, post processor doesn't overwrite this. I assumed the post-processor created the README via synthtool: https://github.com/googleapis/google-cloud-java/blob/0f71f527a7e5c9812767b3c9e0820bc033b57510/generation/new_client/new-client.py#LL223C12-L223C12

If not, how was the README.md created?